pkg/client/unversioned/client_test.go

```diff
@@ -48,7 +48,7 @@ func TestGetServerVersion(t *testing.T) {
 	}))
 	client := NewOrDie(&Config{Host: server.URL})
 
-	got, err := client.ServerVersion()
+	got, err := client.Discovery().ServerVersion()
```

Comment:
lol, that todo is also left by me. I'll remove that TODO. I think the move we did here is in alignment with the client [package structure proposal](https://github.com/kubernetes/kubernetes/blob/master/docs/proposals/client-package-structure.md), so it should be good.